cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/mongo/MongoBackendImpl.java

```diff
@@ -107,7 +107,7 @@ public void createCollection(String dbName, String collectionName, long dataExpi
         try {
             db.createCollection(collectionName);
         } catch (Exception e) {
-            if (e.getMessage().contains("collection already exists")) {
+            if (e.getMessage().contains("\"code\" : 48")) {
```

Comment:
Maybe you have in some place of your doc something like "This sink works for MongoDB 3.2". In that case, this piece of documentation should be update to "This sink works for MongoDB 3.2 and 3.4". If you don't have that piece of documentation maybe it should be created.